docs/assets/javascript/elements/uxdot-sidenav.css

```diff
@@ -32,7 +31,7 @@
   position: relative;
   background-color: var(--rh-color-surface-lightest);
   width: auto;
-  height: 100%;
```

Comment:
I think you still need a `height: 100dvh` here.

Before:
![Screenshot 2024-09-04 at 8 58 51 AM](https://github.com/user-attachments/assets/6ec35f69-0215-41ec-8b1b-2157e61bb7bc)
After:
![Screenshot 2024-09-04 at 8 59 06 AM](https://github.com/user-attachments/assets/2d33fea8-ccc1-4a73-84cc-e9ea7b7b6312)

Reply:
how's that now?